Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -48,7 +48,7 @@ namespace Hazel {
 				// profiling output.
 				if (Log::GetCoreLogger()) // Edge case: BeginSession() might be before Log::Init()
 				{
-					HZ_CORE_ERROR("Instrumentor::BeginSession('{0}') when session '{1}' already open.", name, m_CurrentSession->Name);
+					HZ_LOG_ERROR("Instrumentor::BeginSession('{0}') when session '{1}' already open.", name, m_CurrentSession->Name);
```

Comment:
This and all other logs in header files now violate the one-definition-rule, resulting in **undefined behavior**.

There should be one and only one implementation of a function, you should not have an inline function which has two different implementations, which for this it now will because it differs between two different libraries